src/appleseed/foundation/math/beziercurve.h

```diff
@@ -451,6 +617,23 @@ inline void BezierCurve1<T>::split(BezierCurve1& c1, BezierCurve1& c2) const
 
     c2.m_width[0] = wq;
     c2.m_width[1] = Base::m_width[1];
+
```

Comment:
Remove blank line.

Reply:
I think this blank line does not need to be removed?